scripts/validate_packaged_v8_resources.py

```
@@ -0,0 +1,144 @@
+    from defenseclaw.observability.schema_resources import (
+        telemetry_v8_catalog_bytes,
+        telemetry_v8_compatibility_profile_bytes,
+        telemetry_v8_schema_bytes,
+        v7_exporter_selection_bytes,
+    )
+    from defenseclaw.observability.v8_config import _schema_validator
+
+    _schema_validator()
+    for reference in ("observability.yaml", "observability.md"):
+        if not package_root.joinpath("_data/config/v8", reference).read_bytes():
+            raise PackagedV8ResourceError(f"{label} DefenseClaw v8 reference is empty: {reference}")
```

Ruff (0.15.21)

[warning] Avoid specifying long messages outside the exception class

(TRY003)